app/HutchAgent/Services/CrateService.cs

```diff
@@ -157,6 +160,12 @@ public void UpdateMetadata(string pathToMetadata, Models.WorkflowJob job)
     });
     crate.RootDataset.SetProperty("datePublished", DateTime.Now.ToString("yyyy-MM-dd'T'HH:mm:ssK"));
 
+    var license = new CreativeWork(identifier: _license.Uri); // sets @id to the license URI
```

Comment:
only add this entity conditionally if there is an id/url and a creativework in the config.